guides/source/action_view_helpers.md

```diff
@@ -507,166 +717,328 @@ number_with_precision(111.2345)               # => 111.235
+#=> <chop> is also an example
 ```
 
-NB: The output may still contain unescaped '<', '>', '&' characters and confuse browsers.
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-excerpt)
+for more information.
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_tags) for more information.
+#### pluralize
 
-### UrlHelper
+Returns the singular or plural form of a word based on the value of a number.
 
-Provides methods to make links and get URLs that depend on the routing subsystem.
+```ruby
+pluralize(1, 'person') # => 1 person
+pluralize(2, 'person') # => 2 people
+pluralize(3, 'person', plural: 'users') # => 3 users
+```
 
-#### url_for
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize)
+for more information.
 
-Returns the URL for the set of `options` provided.
+#### truncate
 
-##### Examples
+Truncates a given `text` to a given `length`. If the text is truncated, an omission
+marker will be appended to the result for a total length not exceeding `length`.
 
 ```ruby
-url_for @profile
-# => /profiles/1
+truncate("Once upon a time in a world far far away")
+# => "Once upon a time in a world..."
 
-url_for [ @hotel, @booking, page: 2, line: 3 ]
-# => /hotels/1/bookings/1?line=3&page=2
+truncate("Once upon a time in a world far far away", length: 17)
+# => "Once upon a ti..."
 
-url_for @post # given a composite primary key [:blog_id, :id]
-# => /posts/1_2
+truncate("Once upon a time in a world far far away", length: 17, separator: ' ')
+# => "Once upon a..."
+
+truncate("And they found that many people were sleeping better.", length: 25, omission: '... (continued)')
+# => "And they f... (continued)"
+
+truncate("<p>Once upon a time in a world far far away</p>", escape: false)
+# => "<p>Once upon a time in a wo..."
 ```
 
-#### link_to
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-truncate)
+for more information.
+
+#### word_wrap
+
+Wraps the text into lines no longer than `line_width` width.
+
+```ruby
+word_wrap('Once upon a time', line_width: 8)
+# => "Once\nupon a\ntime"
+```
+
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-word_wrap)
+for more information.
+
+### TagHelper
+
+Provides methods to generate HTML tags programmatically.
+
+#### content_tag
+
+Generates an HTML block tag of the specified type to surround the content. You
+can add HTML attributes by passing an `attributes` hash to options.
+
+```ruby
+content_tag(:div, "Hello world!", class: ["strong", { highlight: current_user.admin? }])
+# => <div class="strong highlight">Hello world!</div>
+ ```
+
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag)
+for more information.
```

Comment:
`content_tag` is [legacy syntax](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag) so I think we can delete this section.
```suggestion
```

Reply:
Thank you, good catch!